### DISCUSSION TITLE
DNN skin layout

### DISCUSSION THREAD

**Discussion by DigitalTrading** [dnnsoftware/Dnn.Platform#5910] [General]
Hi,
I have a DNN site with a custom site skin, I have created a welcome page for the user login so I can style and add modules. however the password reset module now sites behind my navigation and banner at the top of the page and all the modules are gone. Is there any way to style the page the password reset is on?
Thanks
Roger

**Comment by Timo-Breumelhof** [comment]
That is very difficult to answer without an example page, but the fist thing to look for is HTML errors in the Skin.

**Comment by DigitalTrading** [comment]
Thanks for your answer. There are no errors on the page as far as I can see. This will show you the issue:
https://www.bandgjenkins.com/welcome/ctl/passwordreset/resettoken/edc11f71-1e3f-4603-9320-23352cb37a89
This is the password reset link we get.
If you look at the welcome page you will see what it should look like:
https://www.bandgjenkins.com/welcome

Any thoughts greatly appreciated.
Roger

**Comment by Timo-Breumelhof** [comment]
Ah, you named the pane in the header ContentPane, right?
That's the name of the default pane DNN uses for these kind of links (amongst other things)

It's better to name that one something like "HeaderPane" and place the pane with id ContentPane below the menu.

BTW, a better place to ask these kind of questions is on the community site :-)

https://dnncommunity.org/forums

**Comment by DigitalTrading** [comment]
Thanks Timo,
Apologies, I will move the question to the community forum.
What is interesting is that the reset password view appears to be removing all the modules on the welcome page. If I try and edit the page I get no add module options. Its like the page is not a page. after I move this to the community I will be asking which file is generating the view to see if I can hardcode styling to this view.
Thanks again.
Roger

**Comment by Timo-Breumelhof** [comment]
Yes that's how DNN handles these kind of links (by design). It essentially renders a "fake page" and uses the ContentPane to place the reset password control in.

**Comment by Timo-Breumelhof** [comment]
No need to move the question, but the forums just have a bit more users answering these kind of questions :-)

**Comment by DigitalTrading** [comment]
Do you know if there is any way to style the page this sits on? are there page files and css files which I can edit manually?

**Comment by Timo-Breumelhof** [comment]
AFAIK there is no way to set what page to use for this functionality.
The only option I know of to style that page is to use our Style Helper with a condition.

https://www.40fingers.net/Products/DNN-Stylehelper

I would use  IfURL='ctl/passwordreset'
 in combination with AddCssFile="[S]/resetpassword.css"

HTH
